content/automate/ManageIQ/System/CommonMethods/Utils.class/__methods__/log_object.rb

```diff
@@ -37,10 +37,61 @@ def self.current(handle = $evm)
             # instance method
             #    ManageIQ::Automate::System::CommonMethods::Utils::LogObject.log(vm, "My Object", @handle)
 
-            def self.log(obj, object_string = 'Automation Object', handle = $evm)
-              handle.log("info", "Listing #{object_string} Attributes - Begin")
+            def self.log(obj, log_prefix = 'Automation Object', handle = $evm)
+              handle.log("info", "Listing #{log_prefix} Attributes - Begin")
               obj.attributes.sort.each { |k, v| handle.log("info", "   Attribute - #{k}: #{v}") }
-              handle.log("info", "Listing #{object_string} Attributes - End")
+              handle.log("info", "Listing #{log_prefix} Attributes - End")
+            end
+
+            def self.ar_object?(obj)
```

Comment:
@billfitzgerald0120 
can we change this whole method to
```ruby
def self.ar_object?(obj)
   obj.respond_to?(:object_class)
end
```

Reply:
@mkanoor The changes worked so I am updating PR. Thanks